streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedStreamAggregateBuilder.java

```diff
@@ -255,10 +254,11 @@ private void processRepartitions(final Map<KGroupedStreamImpl<K, ?>, Aggregator<
             builder);
     }
 
+    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
```

Comment:
Suppressing the deprecation warnings on our internal processors, and their tests, makes up the lion's share of this PR. I've marked them all with this comment, and filed https://issues.apache.org/jira/browse/KAFKA-12939 to make sure we really migrate everything.